app-ios/Sources/TimetableFeature/TimetableGridCard.swift

```diff
@@ -7,13 +7,16 @@ import CommonComponents
 public struct TimetableGridCard: View {
     let timetableItem: TimetableItem
     let onTap: (TimetableItem) -> Void
+    let cellCount: Int
     
     public init(
         timetableItem: TimetableItem,
+        cellCount: Int? = 1,
```

Comment:
I think this argument is no needed an optional 🤔
and required property is no needed a default value 👍
```suggestion
        cellCount: Int,
```